src/duplicacy_gcdstorage.go

```diff
@@ -45,20 +45,21 @@ type GCDConfig struct {
 	Token        oauth2.Token    `json:"token"`
 }
 
+func (storage *GCDStorage) computeInitialBackoff() float64 {
```

Comment:
It would be great to have a comment explaining the algorithm we're using here and what a user can expect out of Duplicacy in terms of retry timing. The code is pretty simple, but what it's actually going to accomplish isn't intuitively obvious, so a description of the intent is probably warranted here.

Reply:
I will provide one tonight